README.md

```diff
@@ -13,3 +13,78 @@ Editorial changes that improve the readability of the spec or correct spelling o
 
 Please read [CONTRIBUTING.md](CONTRIBUTING.md), about licensing contributions.
 
+Because
+persons and organizations with different levels of
+technicality must be able to contribute, we accept both
+raw files via Pull requests and also propose email proposals.
+
+
+## Why are there different translations?
+
+An ebook can be purchased in any country without limited
+availability or additional delivery costs.
+Readers want consistent display of accessibility
+information, and that’s the primary role of the display
+guide.
+
+The wording proposed in this guide has been widely discussed
+by a large group representing different
+actors of the English-speaking geographies. It has been
+improved after proof-of-concept implementations
+and reviewed by panels of testers. 
+            
+Different countries or different target
+audience services may want to use specific language, and we
+have provided flexibility in the guide to accommodate this
+while maintaining a high level of understanding and similar
+quality to help users in a country find likewise information
+between two bookstores or libraries in the same language
+area.
+
+Simply translating the strings is not enough; the subtle
+meaning of the words and phrases of accessibility concepts
+must be localized for maximum understanding. Therefore, we
+intend to provide a mechanism for the publishing community
+to provide translations that localizes the strings for
+equally effective communication in many languages.
+
+
+With the rapidly evolving landscape of accessible ebooks
+available, most concerned persons are discovering a new
+world. To ensure the information is widely spread and
+understood, the implementation by local actors like
+libraries serving persons with disabilities and other
+local actors in the fight to end the book famine for
+readers with print disabilities is crucial and should be
+highlighted.
+
+The localization of the display guide is a good
+opportunity to make known the rich accessibility
+features offered by modern formats like EPUB. In
+addition, having a local project scale to write an
+understandable vocabulary to describe such features is a
+precious qualitative approach that will benefit every
+player in the value chain.
+
+## How to choose between localization files?
+
+The first keys of each JSON files contains descriptive
+information about it, including:
+
+<ul>
+<li>Author, name of the organisation responsible for the
+establishment and maintenance of this localization
+</li>
+<li>Language, is a 4 letters code where the two first
+letters specifies the language as per ISO 639-1 and
+the two others the country as per ISO 3166-1 alpha-2
```

Comment:
We should also mention it can define different scripts, not just regions.

Reply:
may I let you find the correct wording?

Reply:
See proposal in [Matt suggestion](https://github.com/w3c/publ-a11y-display-guide-localizations/pull/8/commits/1a22914de696382f8dc0b603507b56aff1958199)